rest/config_test.go

```diff
@@ -2619,3 +2599,15 @@ func TestCollectionsValidation(t *testing.T) {
 		})
 	}
 }
+
+func assertX509UnknownAuthority(t testing.TB, actual, expected error) {
```

Comment:
I'd also tweak the name to make it account for the non-x509 error handling we're doing, and leave a comment describing how it covers both (and why we even need this function in the first place, due to the OS-specific errors)

```suggestion
// requireErrorWithX509UnknownAuthority ... TODO
func requireErrorWithX509UnknownAuthority(t testing.TB, actual, expected error) {
```